webviews/src/styles.css

```
@@ -182,12 +186,275 @@ body {
+.base-picker-button[aria-expanded="true"] {
+  border-color: color-mix(in lab, var(--cmux-diff-fg) 30%, transparent);
+  background: color-mix(in lab, var(--cmux-diff-fg) 15%, transparent);
+}
+.base-picker-button:disabled {
+  cursor: default;
+  opacity: 0.85;
+}
+.base-picker-button svg {
+  width: 12px;
+  height: 12px;
+  stroke: currentColor;
```

Stylelint (17.13.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)